### DISCUSSION TITLE
DV-Multiplayer v0.1.8.2 Beta

### DISCUSSION THREAD

**Discussion by AMacro** [AMacro/dv-multiplayer#24] [Announcements]
# Changes / Fixes
## UI
- Fixed issue connecting to games in the server browser where no password is required
- Server information can now be typed on multiple lines
- Dummy servers have been removed from the server browser. The server browser now quieries the lobby server immediately

## Game
- Jobs no longer despawn in single player games when the player is in the station. This extends to multi player games when only one player is in a station. (#22 Jobs De-spawn whilst still in station)
- Jobs now get synced with clients, however, only the host can take jobs at this stage.

# Known Bugs
- Clients are unable to self service locomotives - self service objects (e.g. pipes, chutes, plugs, nozzles) not synced
- Horns, whistles and steam loco reversers can play up when more than one player is in the loco; this is not a breaking issue, only a mild inconvenience.
- Shop, world items, inventory for players other than the host not yet synced
- Many other minor bugs and inconveniences, which we'll work through as we tick off the bigger issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/AMacro/dv-multiplayer/releases/tag/v0.1.8.2-Beta'>DV-Multiplayer v0.1.8.2 Beta</a>.</em>

**Comment by huangzhanboMichael** [comment]
Hello, I am a player in China. In the process of using mod, there is often **a high delay problem**, how should I solve it? In other words, can we use plug-ins to play on the local area network? Looking forward to your reply

**Reply by AMacro** [reply]
Hey @huangzhanboMichael,

If you want to plan on LAN you can use a direct connection, just enter the IP address or name of the computer hosting the game.

If you're wanting to play on the internet, avoid using VPNs, if possible - have the host use port forwarding, if possible for IPv4. If the host is behind CGNAT, IPv6 can be used (need to open the port in the firewall of the router using the host's static IP. 

Services such as Hamachi should also work but may/may not have ping issues.
